src/types/EmbeddedPaymentElement.tsx

```diff
@@ -0,0 +1,427 @@
+ */
+export interface PaymentOptionDisplayData {
+  /**
+   * An image representing the payment method, such as a VISA logo or Apple Pay icon.
+   */
+  image: ImageSourcePropType;
+  /**
+   * A user-facing label for the payment method, like "Apple Pay" or "•••• 4242" for a card.
+   */
+  label: string;
+  /**
+   * Optional billing details associated with the payment method, such as name, email, or address.
+   */
+  billingDetails?: BillingDetails;
+  /**
+   * A string identifier for the type of payment method.
+   * Stripe values: https://stripe.com/docs/api/payment_methods/object#payment_method_object-type
+   * External methods: https://stripe.com/docs/payments/external-payment-methods?platform=ios#available-external-payment-methods
+   * Apple Pay: "apple_pay"
+   */
+  paymentMethodType: string;
+  /**
+   * Optional base64-encoded regulatory mandate text.
+   * If `embeddedViewDisplaysMandateText = false`, this must be shown near the buy button in a tappable view that supports links.
+   */
+  mandateText?: string;
+}
+
+/**
+ * Describes the action performed when the bottom button in the embedded payment form sheet is tapped.
+ * The embedded view may show payment method options such as "Card". When selected, a form sheet appears
+ * for customers to input their payment details. At the bottom of that form sheet is a button.
+ * This type determines what tapping that button does:
+ * - In the `confirm` case, the button says “Pay” or “Set up” and triggers confirmation of the payment or setup intent inside the sheet.
+ * - In the `continue` case, the button says “Continue” and simply dismisses the sheet. The payment or setup is then confirmed outside the sheet, typically in your app.
+ */
+export type EmbeddedFormSheetAction =
+  | {
+      /**
+       * The button says “Pay” or “Set up”. When tapped, it confirms the payment or setup directly within the form sheet.
+       * @param result - Callback invoked with the result of the confirmation. You can use this to show a success message or handle errors.
+       */
+      type: 'confirm';
+      onFormSheetConfirmComplete?: (
+        result: EmbeddedPaymentElementResult
+      ) => void;
+    }
+  | {
+      /**
+       * The button says “Continue”. When tapped, the form sheet closes without confirming anything.
+       * Use this when you want to handle confirmation elsewhere in your app after the customer has filled in their details.
+       */
+      type: 'continue';
+    };
+
+/**
+ * Describes how row selections are handled in the `EmbeddedPaymentElement` view.
+ *
+ * When customers select a payment option (like a card), your app can either:
+ * - Wait for the customer to manually confirm by tapping a button (`default` behavior), or
+ * - Immediately perform an action as soon as the option is selected (`immediateAction` behavior).
+ */
+export type RowSelectionBehavior =
```

Comment:
I _think_ this will need to be passed in `init` for Android, rather than configuration. API is still WIP, so won't know for sure for a little bit. Might be worth keeping this out of the initial implementation to unblock.

Reply:
What's the ETA on this?

Reply:
Tian is on run this week, has a few things to do next week, then will start on the Android API review. So probably 2 ish weeks.